symengine/series.h

```diff
@@ -187,7 +188,7 @@ class SeriesBase : public SeriesCoeffInterface
                                      unsigned int prec)
     {
         if (s == 0)
-            throw std::runtime_error("Series::series_invert: division by zero");
+            throw DivisionByZero();
```

Comment:
Can you change `DivisiionByZero` so that a message like `"Series::series_invert: division by zero"` can be stored with the current message as the default?